components/Image.tsx

```diff
@@ -0,0 +1,3 @@
+import Image from 'next/image';
+
```

Comment:
Can you just add a comment that this is for future accessibility considerations.